package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nestjs-gcp-pubsub",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
no need to add this to a commit. Releases are made separately

Reply:
okay, fixed